docs(Quay): update user doc (#353)
janus-idp/backstage-plugins: plugins/quay/README.md

# Quay plugin for Backstage

The Quay plugin displays the information about your container images within the Quay registry in your Backstage application.

# Table of contents

1. [For administrators](#for-administrators)

   a. [Installation](#installation)

   b. [Development](#development)

1. [For users](#for-users)

>    a. [Using Quay plugin in Backstage](#using-quay-plugin-in-backstage)

Reviewer: It's not clear to me whether this article was omitted on purpose or not:
```suggestion
   a. [Using the Quay plugin in Backstage](#using-quay-plugin-in-backstage)
```

Author: Added the article.

```
yarn workspace @janus-idp/backstage-plugin-quay run start
```

## For users

> ### Using Quay plugin in Backstage

Reviewer: Same as in my previous comment above:
```suggestion
### Using the Quay plugin in Backstage
```

Author: Done

> Quay is a frontend plugin that enables you to view the information about the container images.

Author: Done

#### Prerequisites

- Your Backstage application is installed and running.

> - You have installed the Quay plugin.
> 
>   For installation process, see [Installation](#installation).

Reviewer: If these two sentences belong to one bullet point, then I recommend putting them on the same line, which you can then even shorten, for example:
```suggestion
- You have installed the Quay plugin. See [Installation](#installation).
```

Author: Done

#### Procedure

1. Open your Backstage application and select a component from the **Catalog** page.
1. Go to the **Image Registry** tab.

   The **Image Registry** tab in the Backstage UI contains a list of container images and related information, such as **TAG**, **LAST MODIFIED**, **SECURITY SCAN**, **SIZE**, **EXPIRES**, and **MANIFEST**.

   ![quay-tab](./images/quay-plugin-backstage1.png)

1. If a container image does not pass the security scan, select the security scan value of the image to check the vulnerabilities.

   ![quay-tab](./images/quay-plugin-backstage2.png)

   The vulnerabilities page displays the associated advisory with a link, severity, package name, and current and fixed versions.

   ![quay-tab-vulnerabilities](./images/quay-plugin-backstage3.png)

>    The advisory link redirects to the Red Hat Security Advisory page containing detailed information about the advisory, including the solution.

Reviewer: Just a nitpick, this would make it easier for me to read:
```suggestion
   The advisory link redirects to the Red Hat Security Advisory page that contains detailed information about the advisory, including the solution.
```

Author: Done
